app/lib/ipc.js

```diff
@@ -0,0 +1,32 @@
+const ipc = require('node-ipc');
```

Comment:
can this be solved with electrons ipc?

Reply:
Ok, even with electron native dialogs IPC is needed (with windows and linux). we start a new balloon process when double-clicking associated files (*.burl) but dialogs can only be shown if the app is ready, so we have to inform the main process to show the dialog